src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -324,7 +294,7 @@ const formConfig = {
                 items: {
                   type: 'object',
                   properties: {
-                    treatments: treatmentsSchema
+                    vaTreatments
```

Comment:
I forget--does the new 526 `/submit` schema call it `vaTreatments`?

Reply:
Still `treatments` in the schema  :)